src/pyhf/constraints.py

```diff
@@ -116,10 +110,9 @@ def make_pdf(self, pars):
         if self.batch_size is None:
             normal_means = normal_means[0]
 
-        result = prob.Independent(
+        return prob.Independent(
             prob.Normal(normal_means, self.sigmas), batch_size=self.batch_size
         )
-        return result
 
     def logpdf(self, auxdata, pars):
```

Comment:
i didn't notice it before, but `logpdf` could be rewritten to just be `logpdf(self, pdf, auxdata)` since the only thing that `pars` is used for is to make the pdf `self.make_pdf(pars)`.

Reply:
yes, eventually this method would get dropped like the methods for _ConstraintModel and _MainModel. However I wanted this PR to be pure refactor without changing tests, where this signature is used